doc-content/enterprise-only/fuse/installing-on-fuse-eap-proc.adoc

```diff
@@ -1,12 +1,18 @@
 [id='installing-on-fuse-eap-proc']
-= Installing {FUSE} in {EAP} with {PRODUCT}
-Install {FUSE} in {EAP} with {PRODUCT} to deploy integrated services where required.
+= Installing {FUSE} on {EAP_LONG}
+Install {FUSE_LONG} 7.6 on {EAP} 7.2.3 to use with {PRODUCT} to deploy integrated services where required.
+
+[NOTE]
+====
+{FUSE_LONG} 7.6 is supported on {EAP} 7.2.3, whereas {PRODUCT} is supported on {EAP} {EAP_VERSION}. To ensure a stable deployment environment, install {FUSE} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
+====
 
 .Prerequisites
-* A {PRODUCT} installation on {EAP_LONG} 7.2 is available. For installation instructions, see {URL_INSTALLING_ON_EAP}[_{INSTALLING_ON_EAP}_].
+* A {PRODUCT} installation on {EAP_LONG} {EAP_VERSION} is available. For installation instructions, see {URL_INSTALLING_ON_EAP}[_{INSTALLING_ON_EAP}_].
```

Comment:
Suggestion - add additional prerequisite?

* A separate instance of {EAP_LONG} 7.2.3 is available.

Reply:
Added. Thank you.